eli5/lightgbm.py

```diff
@@ -151,6 +149,88 @@ def get_score_weights(_label_id):
      )
 
 
+@explain_prediction.register(lightgbm.LGBMClassifier)
+@explain_prediction.register(lightgbm.LGBMRegressor)
+def explain_shap_prediction_lightgbm(
```

Comment:
Same here, `explain_shap_prediction_ligthgbm` and `explain_prediction_lightgbm` shares a lot of lines of code. Avoid the duplication of code as suggested for xgboost.